contracts/Governance/GovernorBravoInterfaces.sol

```diff
@@ -139,6 +130,8 @@ contract GovernorBravoDelegateStorageV1 is GovernorBravoDelegatorStorage {
         /// @notice Flag marking whether the proposal has been executed
         bool executed;
 
+        uint8 proposalType;
```

Comment:
Shouldn't this go after the `receipts` mapping?

Reply:
Not sure what you mean

Reply:
I think that proposalType will be packed in the same storage slot with the booleans above and will be okay.
@defcon022  please share your thoughts on that

Reply:
agree with you! okay I will move it